terraform/modules/matrix-service/infra/vpc.tf

```diff
@@ -0,0 +1,137 @@
+//
```

Comment:
Limited experience with VPCs, @mckinsel can I get your review on this?

Reply:
Looks fine to me. Probably boilerplate terraform for a VPC with public subnets. I'm not sure why we want a subnet for each AZ, but maybe something nice happens with Batch failing over.